### PR TITLE
[FIXED JENKINS-38850] - Pass params to ParametersAction to fix shouldSchedule()

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Just a quick bugfix with a set of direct unit tests for the related functionality. The issue itself comes from the partially correct fix of JENKINS-36124 in #100, which was a SECURITY-170 fix attempt.

Functional tests are available in #106

https://issues.jenkins-ci.org/browse/JENKINS-38850

@reviewbybees @cohencil @t0r0X @aheritier

### CODE REVIEW

File: src/test/java/com/tikal/jenkins/plugins/multijob/test/MultiJobParametersActionTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import org.jvnet.hudson.test.Issue;
+
+/**
+ * Tests for {@link MultiJobParametersAction}
+ * @author Oleg Nenashev
+ */
+public class MultiJobParametersActionTest {
+    
+    @Test
+    public void ShouldMergeEmptyParameters() {
```
Comment (jglick) [inline src/test/java/com/tikal/jenkins/plugins/multijob/test/MultiJobParametersActionTest.java]: Why the C#-style casing?

**Comment by oleg-nenashev** [comment]
My fix is probably incomplete for parameters loaded from the disk.
but it should not impact the queue logic anyway.... at least until MultiJob supports Pipeline.
Will fix it anyway.

**Comment by oleg-nenashev** [comment]
@reviewbybees done
@hagzag PTAL, it seems to be a serious regression though there is not so much votes in JIRA

**Comment by oleg-nenashev** [comment]
@hagzag gentle ping

**Comment by hagzag** [comment]
@oleg-nenashev @cohencil is already on this PR, we were planning on releasing last week, I hope we get to it this week. I will keep you posted (or you will get notified via this thread). Sorry about the inconvenience.

**Pull request merged**